mungegithub/mungers/submit-queue.go

```diff
@@ -873,16 +972,16 @@ func (sq *SubmitQueue) getGithubE2EStatus() []byte {
 	return sq.marshal(status)
 }
 
-func (sq *SubmitQueue) getGoogleInternalStatus() []byte {
+func (sq *SubmitQueue) getHealth() []byte {
```

Comment:
this is an existing method, but I can fix this. git was just confused by removing the `getGoogleInternalStatus`, most of serve* methods use one of these get* methods.